Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/PredictorCorrector.java

```diff
@@ -122,7 +206,9 @@ protected double[] getExpectedPoint(DiVector diVector, int position, int baseDim
                 // largest contributor is only 10 percent; there are too many to predict
                 return null;
             }
-            while (pick < baseDimension && values[baseDimension - pick - 1] >= 0.1 * sum) {
+
+            double threshold = min(0.1 * sum, 0.1);
```

Comment:
If the second dimension's attribution is "too small", then should we investigate that dimension? If we trust the attribution from the model (but the model can be wrong) then the model is telling us "where to look". What was a problem was that we originally had 0.1 * sum and the sum came in as (almost) 7.0 which is almost the highest possible RCF score for that setting. The second dimension had a contribution of 0.6 and was not searched. But 0.6 from a single dimension is a large contribution because the long term calibration of (default, inverse expected depth) RCF scores is 1.0 across all dimensions and all shingled components.